tool/rubocop-bundle/Gemfile

```diff
@@ -0,0 +1,18 @@
+source 'https://rubygems.org'
```

Comment:
Minor, but the name _rubocop-bundle_ matches the name pattern for other Rubocop plugins so I thought this was a new tool and not just the _Gemfile_ for our Rubocop setup.